### DISCUSSION TITLE
How to generate Automation Test Report in Detox?

### DISCUSSION THREAD

**Discussion by baskarkarthik** [wix/Detox#2890] [Q&A]
Hi Detox Team,
I'm new to Detox, I'm able set detox framework and execute it, I need HTML report should be generated. I tried doing it but no luck. I'm using mocha runner.

I'm running using cmd 
`>detox test --configuration ios`

I'm trying to generate Allure report it can be generated through 'mocha' commands as below
`>mocha --reporter mocha-allure-reporter`

I'm using detox command not sure how to link mocha with detox cmds for generating Allure report.

Kindly help me to fix this or let me know if any other report available for detox

**Comment by noomorph** [comment]
We don't support the Allure reporter, sorry.
The current integration Jest+Allure is incompatible with `jest-circus` (https://github.com/zaqqaz/jest-allure). 
As for the Mocha, try setting up that reporter in mocha RC file, not via the command line. Maybe that helps, not sure.

**Reply by d4vidi** [reply]
@noomorph is it not possible to have `--reporter ...` being passed-through to mocha by the detox CLI?

**Comment by noomorph** [comment]
Hehe, possible. I already forgot :)))
